ext/pg_query/extconf.rb

```diff
@@ -4,7 +4,7 @@
 require 'mkmf'
 require 'open-uri'
 
-$objs = Dir.glob(File.join(__dir__, '*.c')).map { |f| f.gsub('.c', '.o') }
+$objs = Dir.glob(File.join(__dir__, '*.c')).map { |f| f.gsub(/.c$/, '.o') }
```

Comment:
Curious, can this use #sub, now? We expect 1 match, right?

Reply:
Right, we only expect 1 match here, #sub should be better.

Reply:
Going further, _perhaps_ Pathname#sub_ext supports this kind of operation at another level of abstraction?

https://rubyapi.org/2.4/o/pathname#method-i-sub_ext